indico/modules/events/registration/fields/simple.py

```diff
@@ -226,3 +231,38 @@ class EmailField(RegistrationFormFieldBase):
     @property
     def validators(self):
         return [IndicoEmail()]
+
+
+class AccommodationField(RegistrationFormFieldBase):
```

Comment:
captions should be stored in unversioned data just like we now do for the radio field